lib/lookup.js

```diff
@@ -91,6 +91,10 @@ function resolve(context, next) {
         context.emit('info','lookup-replace',url);
         context.module.raw = url;
       }
+      if (rep.install) {
+        context.emit('info','lookup-install',rep.install);
```

Comment:
also, I think this is something that should be emitted as verbose not info. Thoughts?